project/plugins.sbt

```diff
@@ -8,4 +8,4 @@ addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.5.2")
 
 addSbtPlugin("com.geirsson" % "sbt-ci-release" % "1.5.3")
 
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "2.0.1")
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "2.0.1")
```

Comment:
2.0.1 release was 5 years ago... What do you think about updating to the latest 2.2.1?